pkg/validators/test_am0007.go

```diff
@@ -20,10 +22,63 @@ func (v TestAM0007) Run(mb utils.MetaBundle) (bool, string, error) {
 }
 
 func (v TestAM0007) SucceedingCandidates() []utils.MetaBundle {
-	return testutils.DefaultSucceedingCandidates()
+	csvObj, err := yamlToDynamicObj("../validators/assets/am0007/csv.yaml") // from the pov of validate/validate_test.go
+	if err != nil {
+		panic(err)
+	}
+
+	succeedingCandidates := []utils.MetaBundle{
+		{
+			AddonMeta: &v1alpha1.AddonMetadataSpec{
+				InstallMode: "OwnNamespace",
+			},
+			Bundles: []registry.Bundle{
+				*registry.NewBundle("random-bundle", &registry.Annotations{}, &csvObj),
+			},
+		},
+		{
+			AddonMeta: &v1alpha1.AddonMetadataSpec{
+				InstallMode: "AllNamespaces",
+			},
+			Bundles: []registry.Bundle{
+				*registry.NewBundle("random-bundle", &registry.Annotations{}, &csvObj),
+			},
+		},
+	}
+	return append(succeedingCandidates, testutils.DefaultSucceedingCandidates()...)
 }
 
 // not implemented
 func (v TestAM0007) FailingCandidates() []utils.MetaBundle {
-	return []utils.MetaBundle{}
+	csvObj, err := yamlToDynamicObj("../validators/assets/am0007/csv.yaml")
+	if err != nil {
+		panic(err)
+	}
+
+	return []utils.MetaBundle{
+		{
+			AddonMeta: &v1alpha1.AddonMetadataSpec{
+				InstallMode: "something",
+			},
+			Bundles: []registry.Bundle{
+				*registry.NewBundle("random-bundle", &registry.Annotations{}, &csvObj),
```

Comment:
Great idea. Thinking we could have a `testutils.NewBundle(<path_to_csv>)` helper, given a path, and it can `log.Fatal` like `testutils.DefaultSucceedingCandidates` on errors. WDYT?

Reply:
Yep, sounds good. Done :)